### PR TITLE
Checking Vagrant Version and Plugins

### PR DESCRIPTION
**Pull request by ynixon**
#157 #164 
checked on dev and stage from windows machine with the latest vagrant - 2.2.4

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -4,31 +4,27 @@
 require 'yaml'
 AWS = YAML.load_file 'aws.yml'
 
+if Vagrant::VERSION < '2.2.4'
```
Comment (lioramilbaum) [inline Vagrantfile]: Using the following will be shorter

Vagrant.require_version ">= 2.2.4"

File: Vagrantfile [hunk 2]
```diff
@@ -4,31 +4,27 @@
...
-        # needed for windows as prerequisite for vagrant-aws but will work from vagrant next release 2.2.4
-        if Vagrant::VERSION >= '2.2.4'
+    if ARGV[1] != 'dev' # aws plugin is needed only for non dev environment
+        if Vagrant::Util::Platform.windows?
+            # needed for windows as prerequisite for vagrant-aws
             required_plugins = [
             {"fog-ovirt" => {"version" => "1.0.1"}},
             "vagrant-aws"
             ]
             config.vagrant.plugins = required_plugins
```
Comment (lioramilbaum) [inline Vagrantfile]: This line is duplicated. Move it after the if and use it once.

File: Vagrantfile (hunk 1 above)
Comment (ynixon) [reply Vagrantfile]: done

File: Vagrantfile (hunk 2 above)
Comment (ynixon) [reply Vagrantfile]: done

**Comment by lioramilbaum** [comment]
did you check the changes on vagrant lower than 2.2.4?

**Comment by ynixon** [comment]
I have just played with the numbers, to see that the command works

**Pull request merged**